lib/players/human.rb

```diff
@@ -0,0 +1,9 @@
+module Players
+
+  class Human < Player
+    def move(board)
+      return gets
+    end
+  end
```

Comment:
Do we need the `return` keyword here?  Might be a good idea to use `gets.chomp`, too.  Is there a message to the user, something like "Where do you want to go?"